src/ioctls/vcpu.rs

```diff
@@ -293,6 +300,37 @@ impl VcpuFd {
         Ok(())
     }
 
+    /// This initializes an ARM vCPU to the specified type with the specified features
```

Comment:
Please format the doc comment with summary and description. In the description also add the KVM ioctl that the method is using (KVM_ARM_VCPU_INIT). Since this ioctl has a parameter also add the `# Arguments` section.

Same for `set_one_reg`.